crates/uv-client/src/base_client.rs

```diff
@@ -376,11 +378,58 @@ impl<'a> BaseClientBuilder<'a> {
             path_exists
         });
 
+        // Checks for the presence of `SSL_CERT_DIR`.
+        // Certificate loading support is delegated to `rustls-native-certs`.
+        // See https://github.com/rustls/rustls-native-certs/blob/813790a297ad4399efe70a8e5264ca1b420acbec/src/lib.rs#L118-L125
+        let ssl_cert_dir_exists = env::var_os(EnvVars::SSL_CERT_DIR).is_some_and(|dirs| {
+            // Ignore cases like SSL_CERT_DIR="" or SSL_CERT_DIR="   "
+            if dirs.to_string_lossy().trim().is_empty() {
+                warn_user_once!("Ignoring empty `SSL_CERT_DIR`.");
+                return false;
+            }
+
+            // Parse `SSL_CERT_DIR`, with support for multiple entries using
+            // a platform-specific delimiter (`:` on Unix, `;` on Windows)
+            let (existing, missing): (Vec<_>, Vec<_>) =
+                env::split_paths(&dirs).partition(|p| p.exists());
+
+            if existing.is_empty() {
+                let end_note = if missing.len() == 1 {
+                    "The directory does not exist."
+                } else {
+                    "The entries do not exist."
+                };
+                warn_user_once!("Ignoring invalid `SSL_CERT_DIR`. {end_note}");
+                return false;
+            }
```

Comment:
Why don't we show the values like we do for the `!missing.is_empty()` block? (just wondering)